kong.conf.default

```diff
@@ -29,6 +29,17 @@
                                  # Each Kong process must have a separate
                                  # working directory.
 
+#port_map =                      # When running Kong behind layer 4 port mapping
+                                 # (or port-forwarding), e.g. with
+                                 # `docker run -p 80:8000`, you can use this
+                                 # parameter to let Kong know about published
+                                 # port when it differs from the one Kong is
+                                 # listening, e.g. port_map=80:8000, 443:8443`.`
+                                 # It has effect on `X-Forwarded-Port`, stream
```

Comment:
We need to explain in brief what is the effect.

Reply:
Explaining it in brief is really hard. I tried.